tests/cisco_ios/show_tacacs/cisco_ios_show_tacacs.raw

```diff
@@ -0,0 +1,24 @@
+Tacacs+ Server -  public  :
```

Comment:
Need a parsed file to go with the raw file. The parsed file should be the parsed results from this data.
```
parsed_sample:
  - tacacs: 10.1.1.140
  - tacacs: 10.2.1.140
```
That would be as you have the template now; but you will need to adjust based on above comments for the template parsing.

Reply:
Not quite sure about this step. What this parser is required for?

Reply:
This is required for our tox/travis tests. The parsed file is compared against the raw file to ensure that all parsed data would be derived from the corresponding raw file. Look at https://github.com/networktocode/ntc-templates/tree/master/tests/cisco_ios/show_interfaces/ directory and you will notice that the raw and parsed files have corresponding names. The parsed data is what textfsm would return after parsing the corresponding raw data.